server/cmwell-ws/app/logic/CRUDServiceFS.scala

```diff
@@ -634,11 +635,14 @@ class CRUDServiceFS @Inject()(tbg: NbgToggler)(implicit ec: ExecutionContext, sy
 
   def getLastIndexTimeFor(dc: String = Settings.dataCenter, nbg: Boolean, fieldFilters: Option[FieldFilter]): Future[Option[VirtualInfoton]] = {
```

Comment:
`fieldFilters` here are taken from `/meta/sys/dc/` w/o any sanitation. So, not tightly related to this PR, but I think it would be best to have a special command/handler to write the `/meta/sys/dc` infotons, and evaluate qp to `DirectFieldKey` where needed.
Also, maybe think of some validation that 2 syncs from same dc with different `qp`s do not intersect (very low priority for now, maybe we can just document that such `qp`s must not intersect)